mbtools/models.py

```diff
@@ -149,10 +149,27 @@ def __init__(self, etree, lesson_page):
     def location(self):
         return f"{self.lesson_page.location}: Answer Value"
 
-    def html_element(self):
+    def answer_text_html(self):
         answer_text = self.etree.xpath("answer_text")[0]
         return MoodleHtmlElement(answer_text, self.location)
 
+    def response_html(self):
+        if self.etree.xpath("responseformat")[0].text == "1":
+            response = self.etree.xpath("response")[0]
+            if response.text is not None:
+                return MoodleHtmlElement(response, self.location)
```

Comment:
Would it be useful or confusing to have this location string specify somehow that it's the Response HTML? Or maybe we can update the shared location string to be `f"{self.lesson_page.location}: Answer or Response value"`